packages/embarkjs-ens/src/index.js

```diff
@@ -1,5 +1,6 @@
 /* global global require */
-const EmbarkJS = global.EmbarkJS || require('embarkjs');
+let EmbarkJS = global.EmbarkJS || require('embarkjs');
```

Comment:
Do we still need this? I believe `global.EmbarkJS` was a throwback from when we used to run the same code in the VM and the browser. This code is not meant to run without having done the console check for `EmbarkJS` correct? Maybe we should change this for clarity's sake?